src/NUnitFramework/tests/Attributes/TestOrderAttributeTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using NUnit.TestData;
+using NUnit.TestData.TestCaseSourceAttributeFixture;
+using NUnit.TestUtilities;
+
+namespace NUnit.Framework.Tests.Attributes
+{
+    [TestFixture]
+    class TestOrderAttributeTests
+    {
+        [Test]
+        public void CheckOrderIsCorrect()
+        {
+            var testFixt = TestBuilder.MakeFixture(typeof (TestCaseOrderAttributeFixture));
+            testFixt.Sort();// need to manually call this method to mimic real world conditions
```

Comment:
I actually didn't understand what you meant here. What is the method in the assembly creation that doesn't get called? MakeFixture, of course, is only a test utility, so we can easily make it do whatever is needed.